vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/create/DefineExperimentComponent.java

```diff
@@ -1,184 +0,0 @@
-package life.qbic.datamanager.views.projects.create;
```

Comment:
why was this component deleted - is it not needed to create experiments independently of projects?

Reply:
There is no `define Experiment Component` anymore. This was an abstraction so the same could be displayed in multiple dialogs. As it is only needed in one place it is not only in the experiment dialog